ui/client/src/components/agentic-ai/CollaborationHubView.tsx

```
@@ -0,0 +1,714 @@
+          }, 2000);
+        });
+      } catch (err) {
+        console.error("Error communicating with chat API", err);
+      } finally {
+        setIsLiveRequest(false);
+        streamCompleteResolverRef.current = null;
+        try {
+          const res = await axios.get(
+            `/sessions/session.state.get?sessionId=${sessionPayload.sessionId}`,
+          );
+          return res.data.output;
```

Biome (2.4.14)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)